content/6_Advanced/String_Search.mdx

```diff
@@ -99,7 +99,31 @@ vector<int> pi(const string &s) {
 ```
 
 </CPPSection>
+<PySection>
+
+```py
+def solve(s):
+	n = len(s)
+	pi = [0]*n
+	j = 0
+	for i in range(1,n):
```

Comment:
make spacing consistent and space out operators

Reply:
```suggestion
	for i in range(1, n):
```